test/spec/GitSpec.hs

```diff
@@ -0,0 +1,92 @@
+module GitSpec (spec) where
```

Comment:
The module should be named `Control.Biegunka.Source.GitSpec` as it tests the `Control.Biegunka.Source.Git` module specifically.